app/views/generic_files/upload/_alerts.html.erb

```diff
@@ -1,17 +1,10 @@
-<div class="alert alert-info hide" id="success">
-  You have successfully uploaded some of your files.  Either continue to upload or edit use the links below to abandon the rest of your added files.
-  <p>
-  <%= link_to 'Add Descriptions', GenericFilesController.upload_complete_path(@batch_id), class: 'btn' %>
-  </p>
-</div>
-
-<div class="alert hide" id="fail">
-  There was a problem during upload. Please click the &quot;Start Upload&quot; button or <%= link_to 'start over', sufia.new_generic_file_path %>
+<div class="alert alert-warning" id="fail">
+  There was a problem during upload, none of your files uploaded correctly. Please <%= link_to 'start over', sufia.new_generic_file_path %>.
```

Comment:
I thought about that, but I was not certain how to I18N the links.  If you have a good example I could look into it.